Libraries/CameraRoll/RCTCameraRollManager.m

```diff
@@ -9,11 +9,13 @@
 
 #import <CoreLocation/CoreLocation.h>
 #import <Foundation/Foundation.h>
```

Comment:
The Camera Roll native module had to be stood up for mac because there are RNTester pages that use it.   For example, the xmlhttprequest test page uses it to get and set image files from the camera roll.   It was trivial to stand it up.